ui/app/AppLayouts/Chat/popups/PinnedMessagesPopup.qml

```diff
@@ -124,9 +127,26 @@ StatusDialog {
                     }
                 }
 
+                StatusFlatRoundButton {
+                    id: unpinButton
+                    anchors.right: parent.right
+                    anchors.rightMargin: Style.current.bigPadding
+                    anchors.verticalCenter: parent.verticalCenter
```

Comment:
This doesn't work well with when combined with the disabling of show more button: https://github.com/status-im/status-desktop/pull/9552/files#diff-6222ca23facc418cd1072cc595caa21de1e1ca00dbdbdcd3534b7117da295b83R267

https://user-images.githubusercontent.com/47811206/219321375-8c1aa897-e8ad-4a95-9cb8-66b4c9a45c34.mov

Might be better IMO to anchor it on top + Figma margin. Or even better, to be consistent with the chat history we can position it the same way.

Reply:
Yeah... I'm aware of this but... if I anchor it to the top, and the next message is also long, then hovering over the first one will get you to the same situation -> it won't be visible for the first message

Reply:
True, but when you get used to the chat history (probably the most used view) this looks out of place IMO.

Reply:
Alright, I can definitely anchor it to the top but that's not how it is designed in Figma :)

Reply:
Ohh, just found this to solve the case 😄 : https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/💬-Chat⎜Desktop?node-id=11306%3A391461&t=9UH3o7dF47rANgAH-0

Reply:
@alexjba updated and pushed:
![image](https://user-images.githubusercontent.com/5377645/219370365-c7ccca49-44b4-409f-9ad7-1c32d4c8696e.png)